ReservacionDeVuelos/src/screens/SignInScreen.js

```diff
@@ -1,34 +1,123 @@
-import React from 'react';
-import {View, Text, StyleSheet, TouchableOpacity} from 'react-native';
-import contentText from '../utils/Constants';
+import React, {useState} from 'react';
+import {View, Text, Image, StyleSheet, TouchableOpacity} from 'react-native';
 import colors from '../utils/Colors';
+import contentText from '../utils/Constants';
+import FormButton from '../components/FormButton';
+import FormInput from '../components/FormInput';
+import GoogleButton from '../components/GoogleButton';
+import Checkbox from '../components/Checkbox';
+import eye from '../img/ojo.png';
 
 const SignInScreen = ({navigation}) => {
+  const [name, setName] = useState();
+  const [email, setEmail] = useState();
+  const [password, setPasword] = useState();
```

Comment:
```suggestion
  const [name, setName] = useState('');
  const [email, setEmail] = useState('');
  const [password, setPasword] = useState('');
```

Always define the value type on useState